packages/payload/src/bin/migrate.ts

```diff
@@ -32,7 +32,27 @@ type Args = {
 }
 
 export const migrate = async ({ config, parsedArgs }: Args): Promise<void> => {
-  const { _: args, file, forceAcceptWarning, help } = parsedArgs
+  const { _: args, file, forceAcceptWarning: forceAcceptFromProps, help } = parsedArgs
+
+  const formattedArgs = Object.keys(parsedArgs || {})
```

Comment:
Feels like we could simplify this into one chain since the null will get filtered out by the final `Boolean` filter.

Reply:
Updated